src/utils/wallet.js

```diff
@@ -831,6 +873,38 @@ const wallet = {
     storage.setItem('wallet:notification', false);
   },
 
+  /**
+   * Checks if the zero-balance tokens are set to be hidden
+   *
+   * @return {boolean} If the tokens are hidden
+   *
+   * @memberof Wallet
+   * @inner
+   */
+  areZeroBalanceTokensHidden() {
+    return storage.getItem('wallet:hide_zero_balance_tokens') !== false;
```

Comment:
Maybe create a constant for that attribute so we don't have to repeat it on the methods

Reply:
Refactored these strings to a constant on [e52499f](https://github.com/HathorNetwork/hathor-wallet/pull/309/commits/e52499f06acec98c58d416b081819ae55dbfa2db).

Also centralized other storage access keys on this file.